mesonbuild/interpreter/compiler.py

```diff
@@ -750,29 +754,39 @@ def get_argument_syntax_method(self, args: T.List['TYPE_var'], kwargs: 'TYPE_kwa
         return self.compiler.get_argument_syntax()
 
     @FeatureNew('compiler.preprocess', '0.64.0')
-    @typed_pos_args('compiler.preprocess', varargs=(mesonlib.File, str), min_varargs=1)
+    @typed_pos_args('compiler.preprocess', varargs=(str, mesonlib.File, build.CustomTarget, build.CustomTargetIndex, build.GeneratedList), min_varargs=1)
```

Comment:
I can't remember, does `build.GeneratedTypes` exist in main, or just in my "type the build targets" branch?

Reply:
It exists, but under `if T.TYPE_CHECKING:`, I don't think we can use it here?

Reply:
oh, drp. Never mind then